ctp2_code/gs/slic/slicfunc.cpp

```diff
@@ -4243,7 +4241,7 @@ SFN_ERROR Slic_GetNearestWater::Call(SlicArgList *args)
 	}
 
 	SlicSymbolData *sym = args->m_argValue[1].m_symbol;
-	//Outcommented by Martin G�hmann
```

Comment:
These comments and the out-commented code can also go. When we started working on the code we did it without source control due to licence concerns, and the changes were documented with #if defined(ACTIVISON_ORIGINAL) guards. Eventual, we stripped that. But these comments are still in, in without the original code they don't really make sense anymore. And they actually, also clutter the code.

Reply:
Done in a separate commit.